FLock/README.md

```diff
@@ -1,7 +1,15 @@
-# FLock llm-loss-validator
+# 🔒FLock llm-loss-validator
 
 Validator that computes the validation loss for a huggingface-compatible LLM
 
-# Usage
+# ⚡Usage
 
-FLOCK_API_KEY And task_id get from [FLock](https://www.flock.io),And fill in the deploy.yaml. 
\ No newline at end of file
+FLOCK_API_KEY And task_id get from [FLock](https://beta.flock.io/),and fill in the deploy.yaml.
+
+# 📖 Document
+
+[FLock Docs](https://docs.flock.io/ai-co-creation-platform/guidelines)
```

Comment:
Update link to https://docs.flock.io/flock-product/decentralised-ai-training/validator-guide

The current one is to the co-creation platform which might be a little confusing for validators